### PR TITLE
fix --doctest-modules flag; closes #80

### LINKED ISSUE #80: --doctest-module flag is not recognized
Fails PRs on Travis and my laptop:
https://github.com/dev-cafe/parselglossy/blob/master/.travis.yml#L19

I think it's supposed to be `--doctest-modules`.

### PR DESCRIPTION
**Pull request by bast**

### CODE REVIEW

**Comment by bast** [comment]
Hmm. More problems:
```
hypothesis.errors.DeadlineExceeded: Test took 210.97ms, which exceeds the deadline of 200.00ms
```
I get the same error locally.

**Comment by bast** [comment]
I will try to increase the timeout.

**Comment by bast** [comment]
More breakage :man_shrugging:

File: .travis.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ install:
   - pipenv install --dev -e .
 
 script:
-  - pipenv run py.test --cov=parselglossy tests/ --doctest-module
+  - pipenv run py.test --cov=parselglossy tests/ --doctest-modules
```
Comment (robertodr) [inline .travis.yml]: This option can be entirely nuked as we don't use doctest-style tests at all

File: tests/test_atoms.py [hunk 2]
```diff
@@ -142,6 +142,7 @@ def test_list_quoted_str(a, quoting):
 
 
 @given(a=list_of_complex_numbers())
+@settings(deadline=400.0)
```
Comment (robertodr) [inline tests/test_atoms.py:145]: When Hypothesis timeouts happen it usually means that there's something wrong with the tests themselves. With this project, I've seen it happen when the order of tokens in the grammar is "wrong" which results in longer parsing times. Does this happen locally too?

**Review by robertodr — CHANGES_REQUESTED** [review]

File: tests/test_atoms.py (hunk 2 above)
Comment (bast) [reply tests/test_atoms.py:145]: Yes it also happens locally.

File: .travis.yml (hunk 1 above)
Comment (bast) [reply .travis.yml]: OK I will modify the PR to remove that flag.

**Comment by robertodr** [comment]
Solved with #82

**Pull request closed without merging**